perfkitbenchmarker/pkb.py

```diff
@@ -468,9 +470,13 @@ def Main(argv=sys.argv):
       '%s:  %s' %
       (set_name, benchmark_sets.BENCHMARK_SETS[set_name]['message'])
       for set_name in benchmark_sets.BENCHMARK_SETS]
-  sys.modules['__main__'].__doc__ = __doc__ + (
-      '\nBenchmarks (default requirements):\n\t%s' %
-      _GenerateBenchmarkDocumentation())
+  sys.modules['__main__'].__doc__ = (
```

Comment:
Not currently, though I imagine we may eventually want to get things to work on Windows as well. Considering we already have \n scattered throughout the rest of the code, it's not a blocking issue.